src/universal/services/ono/queries/pullrequests/index.js

```diff
@@ -112,18 +109,30 @@ export type RepositoryGraphType = {
   name: string
 }
 
+export const PullRequestSource = {
```

Comment:
Not sure we we are going to use them, but `TAG` and `BOOK` are some different sources.